dotnet/src/webdriver/BiDi/Communication/EventHandler.cs

```diff
@@ -23,17 +23,17 @@
 
 namespace OpenQA.Selenium.BiDi.Communication;
 
-public abstract class EventHandler(string eventName, Type eventArgsType, IEnumerable<BrowsingContext.BrowsingContext>? contexts = null)
+public abstract class EventHandler(string eventName, IEnumerable<BrowsingContext.BrowsingContext>? contexts = null)
 {
     public string EventName { get; } = eventName;
-    public Type EventArgsType { get; set; } = eventArgsType;
+
     public IEnumerable<BrowsingContext.BrowsingContext>? Contexts { get; } = contexts;
 
     public abstract ValueTask InvokeAsync(object args);
```

Comment:
We can make this slightly more strongly-typed.

```suggestion
    public abstract ValueTask InvokeAsync(EventArgs args);
```

Reply:
Fixed